agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -137,13 +153,47 @@ public static void updateMetricPointBuilder(
         throw new IllegalArgumentException("metric data type '" + type + "' is not supported yet");
     }
 
+    pointBuilder.setValue(pointDataValue);
     pointBuilder.setName(metricData.getName());
-
     metricTelemetryBuilder.setMetricPoint(pointBuilder);
 
-    pointData
-        .getAttributes()
-        .forEach(
-            (key, value) -> metricTelemetryBuilder.addProperty(key.getKey(), value.toString()));
+    if (isPreAggregated) {
+      Long statusCode = pointData.getAttributes().get(SemanticAttributes.HTTP_STATUS_CODE);
+      boolean success = getSuccess(statusCode, captureHttpServer4xxAsError);
+      Boolean isSynthetic =
+          pointData
+              .getAttributes()
+              .get(AttributeKey.booleanKey("applicationinsights.internal.is_synthetic"));
```

Comment:
can we use constant here?

Reply:
constant is defined in bootstrap.. can't reuse it.

Reply:
let's add it in AiSemanticAttributes then?